Makefile

```diff
@@ -4,7 +4,7 @@ export GO111MODULE=on
 # npm/build.
 VERSION = $(shell git describe --tags --dirty --always)
 LDFLAGS = -s -X main.version=$(VERSION)
-ZEEKTAG = v3.0.2-brim1
+ZEEKTAG = dev-zeek-runner
```

Comment:
Will update this once https://github.com/brimsec/zeek/pull/23 is merged & we have a new zeek release that has the zeekrunner.